include/tgfx/core/ImageCodec.h

```diff
@@ -85,12 +85,12 @@ class ImageCodec : public ImageGenerator {
   }
 
   /**
-   * Decodes the image with the specified image info into the given pixels. Returns true if the
-   * decoding was successful. Note that we do not recommend calling this method due to performance
-   * reasons, especially on the web platform. Use the makeBuffer() method for better performance if
-   * your final goal is to draw the image.
-   */
-  virtual bool readPixels(const ImageInfo& dstInfo, void* dstPixels) const = 0;
+  * Decodes or scale the image with the specified image info into the given pixels. Returns
+  * true if the decoding was successful. Note that we do not recommend calling this method due
+  * to performance reasons, especially on the web platform. Use the makeBuffer() method for
+  * better performance if your final goal is to draw the image.
```

Comment:
整段函数注释替换为：Decodes the image into the provided pixel buffer using the specified image info. Returns true if decoding succeeds, or false otherwise. If the size in dstInfo does not match the image's size, the codec will attempt to scale the image to fit dstInfo.